src/cmd/manage_nsfs.js

```diff
@@ -227,13 +230,22 @@ async function verify_bucket_owner(bucket_owner) {
             const data = await get_config_data(full_path, show_secrets);
             if (data.email === bucket_owner) {
                 is_bucket_owner_exist = true;
+                is_allow_bucket_creation = data.allow_bucket_creation;
                 account_id = data._id;
             }
         }
     });
 
     if (is_bucket_owner_exist) {
-        return account_id;
+        if (action === ACTIONS.ADD) {
```

Comment:
I find that the following approach is better understood - 
```
if (!is_bucket_owner_exist) {
        throw_cli_error(ManageCLIError.BucketSetForbiddenNoBucketOwner, `bucket owner ${bucket_owner} does 
        not exists`);
}
if (action === ACTIONS.ADD && !is_allow_bucket_creation) {
        throw_cli_error(ManageCLIError.BucketCreationMotAllowed, bucket_owner);
}
return account_id;
```

Reply:
I left the variable `const detail_msg` because I thought it would be easier to understand than directly put it as an argument in the function.